tests/providers/test_ssn.py

```diff
@@ -143,7 +339,10 @@ def test_pt_BR_ssn(self):
 
     def test_pt_BR_cpf(self):
         for _ in range(100):
-            self.assertTrue(re.search(r'\d{3}\.\d{3}\.\d{3}\-\d{2}', self.factory.cpf()))
```

Comment:
I think your editor or some of your tools reformatted this. Can you revert these whitespace changes? (the ones above re: `ssn` are okay)

Reply:
Absolutely

Reply:
I didn't forget about fixing these whitespace changes -- they're coming shortly